### ISSUE TITLE
Pérennité des identifiants et URL

### ISSUE THREAD

**Issue by JulieChataigner** [cnigfr/schema-paysage#26]
Proposition de rajouter dans les définitions
-  pour les identifiants : qu'ils doivent être non significatif et pérenne en plus d'être unique
- pour les URL : il y a beaucoup de champs url dans le standard, il faudrait sûrement insister pour que ces URLs soient pérennes dans le temps
- un URI pour accéder au jeu de données (peut être en métadonnées)

**Comment by DominiqueLaurent** [comment]
A priori, oui pour les 2 dernières remarques.

Plutôt non pour la première remarque : 
- le paysage est dans le cas général des standards CNIG ("photographie à un moment donné")
- la recommandation actuelle est d'assurer l'unicité des identifiants des découpages paysagers par un système d'espace de nommage : l'identifiant global d'un découpage paysager s'obtient en concaténant l'identifiant de l'atlas (qui est signifiant) et l'identifiant local du découpage (souvent un simple entier incrémental).  Pour l'identifiant de l'atlas, on conseille d'indiquer l'emprise et la date de l'atlas => par construction, cet identifiant et les identifiants dérivés  ne seront pas pérennes. Ce système est relativement simple et s'appuie sur les pratiques en cours de façon à minimiser les efforts de mise en conformité des données
- nous n'avons pas identifié de besoin d'identifiants pérennes. Certes, il peut être intéressant de faire des comparaisons entre découpages paysagers entre plusieurs versions d'un atlas (tous les 10 ans en théorie) mais les relations prédécesseur-successeur peuvent s'obtenir par superposition géométrique et/ou à l'aide du nom du découpage paysager. Un atlas contient un nombre limité de découpages paysagers et il ne faut pas surestimer le rôle des identifiants dans ce contexte.

**Comment by GT-CNIG-DDU** [comment]
Je suis d'accord avec @DominiqueLaurent sur l'absence de nécessité de définir des identifiants non significatifs, au contraire l'aspect signifiant me semble à conserver.
Par contre je ne vois pas de définition claire de la construction des identifiants (suivant les classes d'objets) dans le projet de standard.

**Comment by GT-CNIG-DDU** [comment]
La suite dans #45
